docs/development/setup_development.rst

```diff
@@ -130,6 +130,8 @@ following ports:
 * Source Interface: `localhost:8080 <http://localhost:8080>`__
 * *Journalist Interface*: `localhost:8081 <http://localhost:8081>`__
 
+You should use Tor Browser to test web application changes, `see here for instructions <https://docs.securedrop.org/en/release-0.12.2/development/tips_and_tricks.html#using-tor-browser-with-the-development-environment>`.
```

Comment:
ah this should be an internal hyperlink target: see e.g. http://docutils.sourceforge.net/docs/user/rst/quickref.html#internal-hyperlink-targets 

otherwise this LGTM! thanks @drewmassey

Reply:
Pushed a fix; I'm a bit of an rst n00b so want to confirm that this approach doesn't run afoul of the problem here:
https://stackoverflow.com/questions/15394347/adding-a-cross-reference-to-a-subheading-or-anchor-in-another-page